opendb-core/src/main/java/org/openplacereviews/opendb/dto/HistoryDTO.java

```diff
@@ -0,0 +1,114 @@
+package org.openplacereviews.opendb.dto;
+
+import org.openplacereviews.opendb.ops.OpObject;
+
+import java.util.List;
+import java.util.Map;
+
+public class HistoryDTO {
+
+	private Map<List<String>, List<HistoryObject>> objects;
```

Comment:
Better to have List<HistoryEdit>
HistoryEdit:
- UserId
- Object Type
- Object Edit
- Delta changes (json)